client/extensions/woocommerce/state/sites/settings/stripe-connect-account/selectors.js

```diff
@@ -49,6 +49,33 @@ export function getIsDeauthorizing( state, siteId = getSelectedSiteId( state ) )
 /**
  * @param {Object} state Whole Redux state tree
  * @param {Number} [siteId] Site ID to check. If not provided, the Site ID selected in the UI will be used
+ * @return {boolean} Whether we are presently attempting to complete the OAuth connection
+ */
+export function getIsOAuthConnecting( state, siteId = getSelectedSiteId( state ) ) {
+	return get( getRawSettings( state, siteId ), [ 'isOAuthConnecting' ], false );
+}
+
+/**
+ * @param {Object} state Whole Redux state tree
+ * @param {Number} [siteId] Site ID to check. If not provided, the Site ID selected in the UI will be used
+ * @return {boolean} Whether we are presently requesting oauth initialization
+ */
+export function getIsOAuthInitializing( state, siteId = getSelectedSiteId( state ) ) {
+	return get( getRawSettings( state, siteId ), [ 'isOAuthInitializing' ], false );
+}
+
+/**
+ * @param {Object} state Whole Redux state tree
+ * @param {Number} [siteId] Site ID to check. If not provided, the Site ID selected in the UI will be used
+ * @return {boolean} Whether we are presently requesting oauth initialization
```

Comment:
Looks like the return type and description here are incorrect? Should this be a return of `String` and returning the OAuthUrl?

Reply:
Yep! Fixed in https://github.com/Automattic/wp-calypso/commit/77666c56bca1bb9e208934bea5f4769733bc2a8b